cli/internal/fs/turbo_json.go

```diff
@@ -48,14 +48,13 @@ type RemoteCacheOptions struct {
 }
 
 type rawTask struct {
-	Outputs *[]string `json:"outputs,omitempty"`
-
-	Cache      *bool               `json:"cache,omitempty"`
-	DependsOn  []string            `json:"dependsOn,omitempty"`
-	Inputs     []string            `json:"inputs,omitempty"`
-	OutputMode util.TaskOutputMode `json:"outputMode,omitempty"`
-	Env        []string            `json:"env,omitempty"`
-	Persistent bool                `json:"persistent,omitempty"`
+	Outputs    []string            `json:"outputs"`
```

Comment:
changed this from a pointer, because I had trouble using it as a pointer in the new code, and I'm not sure if it needs to be one. I can change it back if it breaks something

Reply:
A comment from #2712 that got deleted seems to indicating this was a pointer so it would get unmarshaled as `nil` instead of `[]string{}`:
```
// We can't use omitempty for Outputs, because it will
// always unmarshal into an empty array, which means something different from nil.
```
I think we don't need to differentiate between the two now that we don't supply default outputs.

Reply:
I had understood that comment to be about `omitempty`, rather than the pointer (which is why i deleted the comment). Also that comment used to be in the Unmarshal method, rather than where the pointer was defined, so that seems to support the theory
https://github.com/vercel/turbo/blob/v1.6.0/cli/internal/fs/turbo_json.go#L215-L219

Reply:
This is all correct, and a good change. Using weird Go implementation details to hack serialization is a smell, and I'm happy to have it gone.